Parser/pegen/pegen.c

```diff
@@ -364,6 +364,14 @@ tokenizer_error(Parser *p)
 void *
 _PyPegen_raise_error(Parser *p, PyObject *errtype, const char *errmsg, ...)
 {
+    if (p->fill == 0) {
+        va_list va;
+        va_start(va, errmsg);
+        _PyPegen_raise_error_known_location(p, errtype, 0, 0, 0, -1, errmsg, va);
```

Comment:
`_PyPegen_raise_error_known_location` only gained `end_lineno` and `end_col_offset` in 3.10+ in GH-25525.